code/modules/mod/mod_theme.dm

```diff
@@ -960,7 +960,82 @@
 		/obj/item/shield/energy,
 	)
 	skins = list(
```

Comment:
all the skins need to set up their flags correctly with the look on the sprite
i.e. the normal and heavy are closed down, so they wouldnt be neck mounted, engineer is partially closed, just copy it from helmets that look similar

Reply:
fair, didn't rly think about that too much. i'll go through it later